scripts/workflows/release/rollback.cjs

```
@@ -22,159 +22,189 @@ function execGit(command, allowError = false) {
-      log("info", "✓ Rollback pushed to main");
-
-      log("info", "Syncing main back to develop...");
-      execGit(
-        "git checkout develop && git pull origin develop && git merge origin/main -m 'chore: rollback sync (main → develop)' --no-edit",
-      );
-      execGit("git push origin develop");
-      log("info", "✓ Develop synced with rolled-back main");
-    } else {
-      log("info", "[DRY-RUN] Would push rollback changes");
+    if (provider === "shell") {
+      console.log(`[${dryRun ? "DRY-RUN" : "SHELL"}] Deleting tag v${targetVersion}...`);
```

ESLint

[error] 'console' is not defined.

(no-undef)